### ISSUE TITLE
Subtitle strange behavior 

### ISSUE THREAD

**Issue by ghaith100994** [SuRGeoNix/Flyleaf#69]
I Have list of videos that some of it has subtitle (online file subtitle)
when I initialize player I add property Changed 
when open the video  and player status is opening then I till the player to open file subtitle if the  video has subtitle

Here is the strange behavior the Video View will not show subtitle until 1.30 minute is passed but if I open the stream then made seek to the current Time 5 second the subtitle will shown

**Comment by SuRGeoNix** [comment]
There are currently issues with the subtitles, I have not test them as much. I will do it though in the upcoming v3.3

**Comment by ghaith100994** [comment]
It's work with me but only when I did this 
```
private async void Player_OpenCompleted(object sender, Player.OpenCompletedArgs e)
{
            if (e.success && e.type == FlyleafLib.MediaType.Video)
            {
                Player.Play();
                await Task.Run(async () =>
                {
                          while (Player.Session.CurTime < 5000000)
                              await Task.Delay(100);
                          Player.Open(myTempFile);

                });
            }
            OnPropertyChanged(null);
}
```

**Comment by SuRGeoNix** [comment]
It should be fixed on v3.3, if not re-open it!
